packages/purpose-process/test/createReversePurpose.test.ts

```diff
@@ -590,4 +849,71 @@ describe("createReversePurpose", () => {
       ])
     );
   });
+  it("should throw organizationIsNotTheDelegatedConsumer when the requester is the Consumer but there is a Consumer Delegation", async () => {
```

Comment:
Is this test case correct? @AsterITA shouldn't we allow both the consumer and the delegate to create and manage their own purposes?